app/views/steps/case/charges/edit.html.erb

```diff
@@ -10,14 +10,34 @@
     <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
 
     <%= step_form @form_object do |f| %>
+
       <%= f.govuk_text_field :offence_name, autocomplete: 'off' %>
 
       <%= f.fields_for :offence_dates do |od| %>
-        <%= od.govuk_date_field :date, maxlength_enabled: true, legend: { size: 'm' } %>
-        <%= od.button t('.remove_button'), type: :submit, value: '1', name: od.field_name(:_destroy, index: od.id),
-                     class: %w[govuk-button govuk-button--warning],
-                     data: { module: 'govuk-button' } if @form_object.show_destroy? %>
+        <% if od.index.zero? %>
```

Comment:
I'm looking at doing this in a simpler and cleaner way, will share the solution 👍